src/integration-test/java/uk/gov/ida/integrationtest/helpers/MatchingServiceAdapterAppRule.java

```diff
@@ -144,29 +165,35 @@ protected void after() {
-            ConfigOverride.config("metadata.hubTrustStore.path", hubTrustStore.getAbsolutePath()),
-            ConfigOverride.config("metadata.hubTrustStore.password", hubTrustStore.getPassword()),
-            ConfigOverride.config("metadata.idpTrustStore.type", "file"),
-            ConfigOverride.config("metadata.idpTrustStore.path", idpTrustStore.getAbsolutePath()),
-            ConfigOverride.config("metadata.idpTrustStore.password", idpTrustStore.getPassword()),
             ConfigOverride.config("metadata.hubEntityId", HUB_ENTITY_ID),
             ConfigOverride.config("metadata.uri", "http://localhost:" + verifyMetadataServer.getPort() + VERIFY_METADATA_PATH),
             ConfigOverride.config("hub.hubEntityId", HUB_ENTITY_ID)
         ).collect(Collectors.toList());
 
+        if (overrideTruststores) {
+            List<ConfigOverride> trustStoreOverrides = Stream.of(
+                ConfigOverride.config("metadata.trustStore.type", "file"),
+                ConfigOverride.config("metadata.trustStore.path", metadataTrustStore.getAbsolutePath()),
+                ConfigOverride.config("metadata.trustStore.password", metadataTrustStore.getPassword()),
+                ConfigOverride.config("metadata.hubTrustStore.type", "file"),
+                ConfigOverride.config("metadata.hubTrustStore.path", hubTrustStore.getAbsolutePath()),
+                ConfigOverride.config("metadata.hubTrustStore.password", hubTrustStore.getPassword()),
+                ConfigOverride.config("metadata.idpTrustStore.type", "file"),
+                ConfigOverride.config("metadata.idpTrustStore.path", idpTrustStore.getAbsolutePath()),
+                ConfigOverride.config("metadata.idpTrustStore.password", idpTrustStore.getPassword())
+            ).collect(Collectors.toList());
+            overrides.addAll(trustStoreOverrides);
+        }
+
         if (isCountryPresent) {
             List<ConfigOverride> countryOverrides = Stream.of(
                 ConfigOverride.config("europeanIdentity.hubConnectorEntityId", HUB_SECONDARY_ENTITY_ID),
                 ConfigOverride.config("europeanIdentity.enabled", "true"),
 
-                ConfigOverride.config("europeanIdentity.aggregatedMetadata.trustAnchorUri", "http://localhost:" + trustAnchorServer.getPort() + TRUST_ANCHOR_PATH),
-                    ConfigOverride.config("europeanIdentity.aggregatedMetadata.metadataSourceUri", "http://localhost:" + metadataAggregatorServer.getPort() + METADATA_SOURCE_PATH),
-                ConfigOverride.config("europeanIdentity.aggregatedMetadata.trustStore.store", countryMetadataTrustStore.getAbsolutePath()),
-                ConfigOverride.config("europeanIdentity.aggregatedMetadata.trustStore.trustStorePassword", countryMetadataTrustStore.getPassword()),
+//                    ConfigOverride.config("europeanIdentity.aggregatedMetadata.trustAnchorUri", "http://localhost:" + trustAnchorServer.getPort() + TRUST_ANCHOR_PATH),
```

Comment:
Do we need to retain these comments?

Reply:
Definitely not, I just forgot to remove them.